modules/docs/arrow-docs/docs/docs/generic/coproduct/README.md

```diff
@@ -37,6 +38,18 @@ fun toDisplayValues(items: List<Coproduct3<Car, Dealership, Salesperson>>): List
     )
   }
 }
+
+fun main() {
+    println(
+        toDisplayValues(
+            listOf<Coproduct3<Car, Dealership, Salesperson>>(
+                Car(Speed(100)).first(),
+                Dealership("Cedar Falls, Iowa").second(),
+                Salesperson("Car McCarface").third()
+            )
+        )
+    )
+}
 ```
 
 Let's say we have an api. Our api operates under the following conditions:
```

Comment:
I think we could also explain here what's the most effective benefit we've found of Coproducts over sealed classes: to flat the hierarchy. The alternative is composed sealed classes, which doesn't leave you in a pretty neat scenario (sealed classes don't support inheritance outside of their own hierarchy). Describing both scenarios and how they differ would be very enlightening for the reader.